integrations/docker/images/stage-2/chip-build-efr32/Dockerfile

```diff
@@ -41,3 +44,13 @@ RUN pip3 install lark jinja2 stringcase
 
 # Sphinx dependencies (for slc-cli)
 RUN pip3 install myst_parser sphinx_rtd_theme sphinx_tabs linkify-it-py
+
+FROM ghcr.io/project-chip/chip-build:${VERSION}
```

Comment:
Please update the PR description to explain changes as this starts being more than "add wiseconnect".

In particular I am unclear bout this from: does this start a new image copying from "build"? If yes, what do the pip32 installs above do because they affect build but not this new version and no python bits are copied over ... so should we delete or move those?

Reply:
@andy31415 these should not affect the matter build since gn would work with the sources, anyway I have made the changes to the Dockerfile.